pkg/cmd/server/api/validation/master.go

```diff
@@ -192,5 +192,12 @@ func ValidatePolicyConfig(config api.PolicyConfig) fielderrors.ValidationErrorLi
 func ValidateProjectRequestConfig(config api.ProjectRequestConfig) fielderrors.ValidationErrorList {
 	allErrs := fielderrors.ValidationErrorList{}
 
+	if len(config.ProjectRequestTemplate) > 0 {
```

Comment:
The sentence following that is misleading :) 'If it is not specified, then projectrequest will not work.'